web/WebDevelopersGuide.md

```diff
@@ -25,6 +25,8 @@ There, they must register with their active Auth Token to gain access to the tab
 interact with a table, the client should first request the details about the table. With those details, the client may request the 
 the contents of the table and subscribe to any updates that occur. 
 
+// TODO Does this need to be changed?
```

Comment:
Sure looks like. Ask @niloc132 to comment.

Reply:
@niloc132 mentioned this file needs to be deleted and he will do it as part of a separate PR, so removing my edit from here.

Reply:
This file should have been deleted long ago, see https://github.com/deephaven/deephaven-core/pull/6342